src/main/java/com/solvd/movie/service/MovieClient.java

```diff
@@ -4,7 +4,7 @@
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
-public interface MovieService {
+public interface MovieClient {
```

Comment:
Internal services should have `Service` suffix, external ones - `Client`

Reply:
Fixed, because this service is internal